app/state/reducers/searchFiltersReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  search: '',
+};
+
+function searchFiltersReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ENTER_OR_CHANGE_SEARCH_PAGE': {
```

Comment:
Yeah, you're right about using handleActions. That will also solve the curlybraces issue.
I changed the search page reducers to use `handleActions`.